### DISCUSSION TITLE
Flashable binaries for new maintenance release ?

### DISCUSSION THREAD

**Discussion by rcmcronny** [Gifford47/HCPBridgeMqtt#75] [General]
@Gifford47 @Tysonpower 
Hi, 

short question, will there be binaries (as for the former release) made available, also ? 
Would appreciate it :)

Thanks Ronny

**Comment by Gifford47** [comment]
Yes, it will be. But the most problem are build options/features. We cannot provide every build for every possible option or combination of sensors.
Therefore i will publish a standard binary which will provide the basic features of a single board and another one with a additional sensor like bme.

**Reply by rcmcronny** [reply]
Thats fully ok for me, if i attach additional sensors, i will compile them myself, till that, i want to use the normal build (with tysonpowers board)  for the garagedoor to keep it simple :)

**Comment by rcmcronny** [comment]
The new update from today contains a esp32.bin file. Is this usable with @Tysonpower 's board ?

**Reply by Gifford47** [reply]
I think not. I don't use this board type as I have my own.
I'm sry but you have to build it on your own or ask tysonpower to provide it.
